src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,35 @@
+@import '../utils/mixins';
+
+.stars-area {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  margin-top: 16px;
```

Comment:
It seems that the BEM block name `stars-area` does not match the file name `stars.scss`. According to BEM methodology and file structure conventions, the block name should match the file name. If `stars-area` is a separate block, it should be in its own file named `stars-area.scss`.